src/main/dynamo/util.ts

```diff
@@ -22,6 +17,7 @@ export async function encryptOrPassThroughItems<T extends Record<string, any>>(
     const fieldsToEncrypt = Object.keys(item).filter(
       (_) => !encryptionBlacklist.has(_)
     )
+    const f = jayz.encryptItem(item, fieldsToEncrypt)
```

Comment:
spurious line?  or did you intend to use `f` below?

Reply:
Gah, nice catch. Was left over from inspecting the inferred types to fix Titi's issue w/ trying to use flatMap with the es2019 lib. Removed in: https://github.com/ginger-io/beyonce/commit/5d3584bc52a31f7f2699ccbeac326b8f25eb42a7